python/ray/_private/accelerators/amd_gpu.py

```diff
@@ -55,8 +67,28 @@ def get_current_process_visible_accelerator_ids() -> Optional[List[str]]:
     @staticmethod
     def get_current_node_num_accelerators() -> int:
         num_gpus = 0
+        major_version = 0
 
-        if sys.platform.startswith("linux"):
+        try:
+            major_version = AMDGPUAcceleratorManager._get_rocm_major_version()
+        except Exception:
+            return 0
+
+        if major_version >= 6:
+            try:
+                amdsmi_init()
```

Comment:
I think we need to handle the case where major version is >= 6 but amdsmi package is not installed?